### ISSUE TITLE
[Bug]: App can't be started more then once on RDS/Citrix

### ISSUE THREAD

**Issue by JtheBAB** [electron/electron#34281]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

18.0.3

### What operating system are you using?

Windows

### Operating System Version

Windows Server 2019

### What arch are you using?

x64

### Last Known Working Electron version

13.2.0

### Expected Behavior

User A, User B, User C and so on can start the App in each session.

### Actual Behavior

- User A starts the App
- Everything is working as expected
- User A lets the App open
- User B starts the App
- Nothing happen. No error, no crash, no exe is started in User B session

### Additional Information

I upgraded Mattermost Desktop App (Build with electron) to the [latest version](https://github.com/mattermost/desktop/compare/v5.0.4...v5.1.0). They changed to a newer electron version. I know that Slack Desktop Apphas the same problem (as far as i know that's also electron).

Basically it is impossible to use Apps build with newer electron versions on Remote Desktop Session Hosts (RDS/RDP) (tested), Citrix Virtual Desktop/Apps (tested) or Azure Windows 10 Multi Device (guessing).

**Comment by ankar84** [comment]
related to https://github.com/RocketChat/Rocket.Chat.Electron/issues/2408

**Comment by ankar84** [comment]
Fix could be in 19 Electron https://github.com/electron/electron/pull/34160

**Comment by JtheBAB** [comment]
Thanks for pointing to that :)

Fix is in Electron 18.2.4. Successfully tested [here](https://github.com/mattermost/desktop/pull/2119)

**Comment by VerteDinde** [comment]
Thanks @JtheBAB, glad it's fixed for you! As a heads up, you may run into a similar deadlock error if you try to run two instances of the same app on the same user profile. This PR should also fix that issue. I'm going to close this issue for now, since it seems to be fixed, but please feel free to open a new issue if you run into more problems 🙇‍♀️
